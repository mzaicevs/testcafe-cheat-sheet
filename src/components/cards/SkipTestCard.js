import React from "react";
import { skippingTests } from "../../snippets";
import MultiCodeCard from "../../base_components/MultiCodeCard";

const SkipTestCard = () => (
  <MultiCodeCard
    cardTitle={skippingTests.title}
    fullDocLink={skippingTests.fullDocLink}
    codeSections={skippingTests.codeSections}
  />
);

export default SkipTestCard;
