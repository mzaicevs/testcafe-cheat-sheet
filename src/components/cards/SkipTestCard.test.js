import React from "react";
import { shallow } from "enzyme";
import SkipTestCard from "./SkipTestCard";

jest.mock("react-syntax-highlighter/dist/esm/styles/hljs", () => () => {});

describe("SkipTestCard", () => {
  it("matches snapshot", () => {
    const wrapper = shallow(<SkipTestCard />);
    expect(wrapper).toMatchSnapshot();
  });
});
