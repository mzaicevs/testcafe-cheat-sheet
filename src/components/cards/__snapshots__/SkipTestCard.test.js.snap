// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkipTestCard matches snapshot 1`] = `
<CodeCard
  cardTitle="Skipping Tests"
  codeSections={
    Array [
      Object {
        "code": Array [
          "fixture.skip",
          "test.skip",
        ],
        "description": "Specify that a particular test or fixture should be skipped when running tests.",
        "language": "javascript",
      },
      Object {
        "code": Array [
          "fixture.only",
          "test.only",
        ],
        "description": "Specify that only a particular test or fixture should run while all others should be skipped.",
        "language": "javascript",
      },
    ]
  }
  fullDocLink="https://devexpress.github.io/testcafe/documentation/test-api/test-code-structure.html#skipping-tests"
/>
`;
